internal/util/sessionutil/session_util.go

```diff
@@ -329,6 +335,21 @@ func (s *Session) getServerID() (int64, error) {
 	return nodeID, nil
 }
 
+func (s *Session) getServerLabelsFromEnv(role string) map[string]string {
+	ret := make(map[string]string)
+	switch role {
+	case "querynode":
+		supportedLabels := paramtable.Get().QueryNodeCfg.LabelAwareQueryNodeBalance.GetAsStrings()
```

Comment:
It's wried that use a configuration to setup the key and get the value from environment.
Use a static prefix-based environment variable may be better? It also prevent from the environment conflict with other software on same host
For example, `MILVUS_SERVER_LABEL_QUERYNODE_LOCATION` set up a label with key `LOCATION` at querynode.